docs/flows-format.adoc

```diff
@@ -9,7 +9,7 @@ The "Filter ID" column shows which related name to use when defining Quick Filte
 
 The "Loki label" column is useful when querying Loki directly: label fields need to be selected using link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].
 
-The "Cardinality" column gives information about the implied metric cardinality if this field is used as a Prometheus label with the `FlowMetrics` API. Refer to the `FlowMetrics` documentation for more information on using this API.
+The "Cardinality" column gives information about the implied metric cardinality if this field was to be used as a Prometheus label with the `FlowMetrics` API. Refer to the `FlowMetrics` documentation for more information on using this API.
```

Comment:
@skrthomas this change is not intentional from me, but happened when I regenerated the doc. I can't remember if the desired formulation is "is used" or "was to be used".

Side remark on the flows format reference: [in the docs](https://docs.openshift.com/container-platform/4.15/observability/network_observability/json-flows-format-reference.html) the page title is "Network flows format reference" but the menu item is "JSON flows format reference". Could we use the former only? It doesn't make much sense today to mention "JSON", because that's not only used in JSON but also in other formats such as prometheus labels and soon open telemetry

Reply:
This is really minor, but "Is used" is preferable just because "was to be" is passive and past tense.

Reply:
ACK, will make sure these match to the "Network flows format reference"